.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on: 
+  push:
+  pull_request:
+jobs:
+  nblint:
+    name: Lint Notebooks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cleanlab/nblint-action@v1
+        with:
+          ignore_paths: 'contrib'
+          check_correct_kernel: true
+          check_notebook_badge: true
+          check_outputs_empty: false
+          check_no_trailing_newline: false
```

Comment:
Do we want to set this to true eventually in a separate PR?  If so, you can make a github issue to track this, marked "good first contribution"?  (I assume it is quite a bit of work to clean up all the notebooks to pass this check which is why it's set to false).

Reply:
I think it doesn't really matter for these notebooks; it matters for the ones in the main cleanlab repo, because the way we're rendering the notebooks to HTML, they look bad if there's a trailing newline.

Reply:
Basically what Anish said! I don't think it's too important because it's mainly only viewed in the GitHub, we can choose to enforce that if we want consistency with the main repo (+ to look slightly nicer perhaps) but i think it's extremely low priority.